libs/components/theme/src/lib/theming/theme.service.spec.ts

```diff
@@ -16,16 +17,19 @@ describe('Theme service', () => {
       current.theme.hostClass
     );
 
-    let addModeClassMatcher = expect(mockRenderer.addClass);
+    let addClassMatcher = expect(mockRenderer.addClass);
 
     if (current.theme.supportedModes.indexOf(current.mode) < 0) {
-      addModeClassMatcher = addModeClassMatcher.not;
+      addClassMatcher = addClassMatcher.not;
     }
 
-    addModeClassMatcher.toHaveBeenCalledWith(
-      mockHostEl,
-      current.mode.hostClass
-    );
+    addClassMatcher = expect(mockRenderer.addClass);
+
+    if (current.theme.supportedSpacing.indexOf(current.spacing) < 0) {
+      addClassMatcher = addClassMatcher.not;
+    }
+
+    addClassMatcher.toHaveBeenCalledWith(mockHostEl, current.spacing.hostClass);
```

Comment:
I don't like this at all--seems like it makes it harder to read/determine what is actually being tested just in the name of saving a couple lines. Like it looks like line 23 doesn't even matter??? We immediately reassign it on 26?